Source/Provers/SMTLib/SMTLibBatchTheoremProver.cs

```diff
@@ -87,7 +88,10 @@ public override int FlushAxiomsToTheoremProver()
         FlushAxioms();
 
         Push();
-        SendVCAndOptions(descriptiveName, vcString);
+        if (this.libOptions.EmitDebugInformation) {
```

Comment:
Can we deduplicate this `if (this.libOptions.EmitDebugInformation) {`

Reply:
Probably so. I'll look into it.